CMakeLists.txt

```diff
@@ -30,6 +30,13 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES ".*Clang")
     endif()
 endif()
 
+find_package(libSquish REQUIRED)
```

Comment:
The case needs to match the Find*.cmake for case-sensitive OSes

Reply:
Good to know.  I wasn't aware it did this search in a case-sensitive manner.